art/attacks/evasion/auto_projected_gradient_descent.py

```diff
@@ -203,6 +203,7 @@ def __call__(self, y_true: tf.Tensor, y_pred: tf.Tensor, *args, **kwargs) -> tf.
                     nb_classes=estimator.nb_classes,
                     input_shape=estimator.input_shape,
                     loss_object=_loss_object_tf,
+                    optimizer=estimator._optimizer,
```

Comment:
Should we use the new property:
```suggestion
                    optimizer=estimator.optimizer,
```